library/Icingadb/Authentication/ObjectAuthorization.php

```diff
@@ -177,7 +177,7 @@ protected function loadGrants(string $model, Filter\Rule $filter, string $cacheK
             }
 
             $inspectedRoles[] = $role->getName();
-            $roleName = $this->getDb()->quoteIdentifier($role->getName());
+            $roleName = $this->getDb()->quoteIdentifier((array) $role->getName());
```

Comment:
Please don't cast it. You never know when PHP decides to transform `'abc'` to `['a', 'b', 'c']` in some future version. (We don't want *the name to be an array*, but *inside an array*)